public/rules.json

```diff
@@ -35,148 +192,192 @@
           "rules": [
             {
               "rule": "<G1>",
-              "description": "Treat everyone with respect",
+              "description": " Treat everyone with respect",
               "link": "https://www.robotevents.com/storage/game_manual/VRC_2023-2024_Over_Under/rules/G1.html"
             },
+
             {
               "rule": "<G2>",
-              "description": "VRC is a student-centered program",
+              "description": " VRC is a Student-centered program",
               "link": "https://www.robotevents.com/storage/game_manual/VRC_2023-2024_Over_Under/rules/G2.html"
             },
+
             {
               "rule": "<G3>",
-              "description": "Use common sense",
+              "description": " Use common sense",
               "link": "https://www.robotevents.com/storage/game_manual/VRC_2023-2024_Over_Under/rules/G3.html"
             },
+
             {
               "rule": "<G4>",
-              "description": "The Robot must represent the skill level of the Team",
+              "description": " The Robot must represent the skill level of the Team",
               "link": "https://www.robotevents.com/storage/game_manual/VRC_2023-2024_Over_Under/rules/G4.html"
             },
+
             {
               "rule": "<G5>",
-              "description": "Robots begin the Match in the starting volume",
+              "description": " Robots begin the Match in the starting volume",
               "link": "https://www.robotevents.com/storage/game_manual/VRC_2023-2024_Over_Under/rules/G5.html"
             },
+
             {
               "rule": "<G6>",
-              "description": "Keep your Robots together",
+              "description": " Keep your Robots together",
               "link": "https://www.robotevents.com/storage/game_manual/VRC_2023-2024_Over_Under/rules/G6.html"
             },
+
             {
               "rule": "<G7>",
-              "description": "Don’t clamp your Robot to the field",
+              "description": " Don’t clamp your Robot to the Field",
               "link": "https://www.robotevents.com/storage/game_manual/VRC_2023-2024_Over_Under/rules/G7.html"
             },
+
             {
               "rule": "<G8>",
-              "description": "Only Drivers, and only in the Alliance Station",
+              "description": " Only Drive Team Members, and only in the Alliance Station",
```

Comment:
You should get rid of the leading space in every rule description. Is that to space it out in the Rules selector?